packages/vue-dot/src/elements/DialogBox/DialogBox.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+	<VDialog
+		:value="dialog"
+		:width="width"
+		@input="$emit('update:dialog', $event)"
+		@keydown.esc="close"
+	>
+		<VCard class="py-6 px-6">
+			<VCardTitle class="pa-0 mb-5">
+				<h2 class="title font-weight-bold">
+					{{ title }}
+				</h2>
+
+				<VSpacer />
+
+				<VBtn
+					icon
+					@click="close"
+				>
+					<VIcon>
+						{{ closeIcon }}
+					</VIcon>
+				</VBtn>
+			</VCardTitle>
+
+			<VCardText>
+				<slot />
+			</VCardText>
+
+			<VCardActions class="pa-0">
+				<VSpacer />
+				<slot name="footer">
+					<VBtn
+						class="ml-6 px-5"
+						@click="$emit('cancel')"
+					>
+						{{ locales.cancelBtn }}
+					</VBtn>
+
+					<VBtn
+						color="primary"
+						class="ml-6 px-5"
+						@click="$emit('validate')"
+					>
+						{{ locales.validateBtn }}
+					</VBtn>
+				</slot>
+			</VCardActions>
+		</VCard>
+	</VDialog>
+</template>
+<script lang="ts">
+	import Vue, { PropType } from 'vue';
+	import Component from 'vue-class-component';
+	import { mdiClose, mdiPhone } from '@mdi/js';
+	import { locales } from './locales';
+
+	const Props = Vue.extend({
+		props: {
+			/**
+			 *  Show dialog or not
+			 *  Default value false, dialog not showing
+			 */
+			dialog: {
+				type: Boolean,
+				default: () => false
```

Comment:
```suggestion
				default: false
```
On utilise les fonctions pour les valeurs par défaut uniquement lorsque nécessaire